frappe/templates/web.html

```diff
@@ -4,14 +4,13 @@
 {% block content %}
 <div class="page-container" id="page-{{ name or page_name }}" data-path="{{ pathname }}"
 	{% if page_or_generator=="Generator" %}data-doctype="{{ doctype }}"{% endif %}>
-		<div class="page-content {% if not show_sidebar %}page-max-width{% endif %}">
 			<div class="row {% if show_sidebar %}vert-line{% endif %}">
 				{% if show_sidebar %}
 					<div class="col-sm-3 sidebar-block">
 						{% include "templates/includes/web_sidebar.html" %}
 					</div>
 				{% endif %}
-				<div class="{% if show_sidebar %}content-block col-sm-9{% else %} content-block col-sm-12 {% endif %}">
+				<div class="{% if show_sidebar %}page-content col-sm-9{% else %} page-content col-sm-12 {% endif %}">
```

Comment:
Why change the class name? I think there is a lot of CSS associated with content-block